frontend/packages/console-dynamic-plugin-sdk/src/extensions/yaml-templates.ts

```diff
@@ -1,12 +1,12 @@
-import { ExtensionK8sModel } from '../api/common-types';
+import { ExtensionK8sGroupKindModel } from '../api/common-types';
 import { Extension, ExtensionDeclaration, CodeRef } from '../types';
 
 /** YAML templates for editing resources via the yaml editor. */
 export type YAMLTemplate = ExtensionDeclaration<
   'console.yaml-template',
   {
     /** Model associated with the template. */
-    model: ExtensionK8sModel;
+    model: ExtensionK8sGroupKindModel;
```

Comment:
It seems odd to me that a yaml template isn't tied a version.

cc @spadgett

Reply:
why not use the dynamic version which is available in the cluster?

Reply:
> why not use the dynamic version which is available in the cluster?

They're not API compatible. Different API versions need different YAML templates due to API differences.

The API version should be part of the model, though, right?